README

```diff
@@ -1 +1,2 @@
 # Description of obiwan-app
+# Features
```

Comment:
Hi. Sorry mate but for now I don't want to add features to README file.

Reply:
It's ok. I'm not gonna add any line to README. :+1: